python/core/looping/using-else-in-loops.md

```diff
@@ -86,17 +87,17 @@ Fill in the following snippet such that it will return *"Computation Done"* when
 
 ```python
 x = 0
-while x ??? 3
+while x ??? 3:
     print("running")
     x += 1
 ???:
     ???("Computation Done")
 ```
 
-* <
+* `<`
```

Comment:
Why wrap only this answer in backticks?

Reply:
Seems to be convention for this particular area of the curriculum, we can re-evaluate another time.